client/src/app/core/repositories/agenda/item-repository.service.ts

```diff
@@ -134,6 +135,19 @@ export class ItemRepositoryService extends BaseHasContentObjectRepository<
         return viewModel;
     }
 
+    /**
+     * The `CollectionStringMapperService` returns the related repository and model-constructor by passing the
+     * underlying collection of the given item.
+     *
+     * @param item is the view-item which is duplicated.
+     */
+    public createCopyOfItem(item: ViewItem): void {
+        const repo = this.collectionStringMapperService.getRepository(item.collection);
+        const ModelConstructor = this.collectionStringMapperService.getModelConstructor(item.collection);
+        const copy = (item.contentObject.getModel() as BaseModel).getCopy();
```

Comment:
You do not need to cody this. Please remove all copy-related code (e.g. `getCopy()`)